include/circt/Dialect/FIRRTL/FIRRTLUtils.h

```diff
@@ -322,6 +322,14 @@ static ResultTy transformReduce(MLIRContext *context, RangeTy &&r,
 /// Truncate `a` to the common prefix of `a` and `b`.
 void makeCommonPrefix(SmallString<64> &a, StringRef b);
 
+//===----------------------------------------------------------------------===//
+// OM dialect utilities
```

Comment:
nit: it's the FIRRTL dialect, but "object" utilities. Maybe call it something like "FIRRTL dialect object utilities" or something like that?